hyperbahn-dev.sh

```diff
@@ -1,14 +1,24 @@
 #!/bin/sh 
 set -e
 
+H0="127.0.0.1:21300"
+H1="127.0.0.1:21301"
+
+if [ ! -d ./node_modules ]; then
```

Comment:
Please `which tcurl` and `npm install tcurl --global` if not tcurl.

Please also update README